models/issues/issue_search.go

```diff
@@ -480,7 +480,7 @@ func IssueIDs(ctx context.Context, opts *IssuesOptions, otherConds ...builder.Co
 	applySorts(sess, opts.SortType, opts.PriorityRepoID)
```

Comment:
```suggestion
```
As a simple fix way, we only need issue ids here, so no need to add sort options.

Reply:
But maybe in some cases, we need a sorted issue ids list? Not sure about this.

Reply:
Thanks for your advice! I also agree that removing `applySorts` will solve the problem, but manually removing the duplicate key looks safer. I doubt if the function `Issues()` will have the same problem as duplicate records too and we just haven't discovered the case yet.